upsertGitHubTag/deployment/index.js

```diff
@@ -253,6 +254,23 @@ function processEvent(event, callback) {
         " from GitHub.",
     });
   }
+  // If bucket name is not null (had to put this for the integration test)
+  if (process.env.BUCKET_NAME) {
+    // Send payload to s3
+    const client = new S3Client({});
+    const command = new PutObjectCommand({
+      Bucket: process.env.BUCKET_NAME,
+      Key: deliveryId,
```

Comment:
This will put all objects in the "root" of the bucket. Is that what we want? If we know the deliveryId, then it works. If we don't know it, then it's going to be hard to find.

Is it worth putting them in keys by date/org or org/date?

I don't know the answer, just raising the question. It depends on how we expect to use this.

Reply:
I had to change it to date/deliveryid as there can be multiple repos in a single delivery.